jboss-kie-optaweb-employeerostering/added/launch/jboss-kie-optaweb-employeerostering.sh

```diff
@@ -16,11 +16,14 @@ function configure() {
 }
 
 function configure_optaweb_employeerostering_persistence() {
-    # for persistence.xml, in order to use external DB instead of the hard-coded default H2 db. 
+    # for persistence.xml property replacement
+    if [ "${OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_DATASOURCE}" != "" ]; then
+        JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.optaweb.employeerostering.persistence.datasource=${OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_DATASOURCE}"
+    fi
     if [ "${OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_DIALECT}" != "" ]; then
         JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.optaweb.employeerostering.persistence.dialect=${OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_DIALECT}"
     fi
-    if [ "${OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_DS}" != "" ]; then
-        JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.optaweb.employeerostering.persistence.datasource=${OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_DS}"
+    if [ "${OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_HBM2DDL_AUTO}" != "" ]; then
```

Comment:
Persistence.ds is not needed?

Reply:
The env var is _DATSOURCE is for employee rostering, to match the system property name that is being set. In the templates we do populate it with the _DS parameter, along with the _JNDI one.

Reply:
got it.